apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -690,7 +739,7 @@ contract DisputableVoting is DisputableAragonApp, IForwarder {
     * @return True if the given voter can participate a certain vote
     */
     function _canVote(Vote storage vote_, address _voter) internal view returns (bool) {
-        return _isVoteOpenForVoting(vote_) && _hasVotingPower(vote_, _voter);
+        return _isVoteOpenForVoting(vote_) && _hasVotingPower(vote_, _voter) && _voteCaster(vote_, _voter) != _voter;
```

Comment:
This means that voters can’t change their cast votes,  right? Some discussion rings a bell about this, but I don’t remember it. What was the rationale for this change?

Reply:
Yes, simply because since multiple extensions are allowed, if we allow voters to change their vote, then they could extend votes indefinitely